packages/cf-component-card/src/CardDrawers.js

```diff
@@ -9,23 +9,38 @@ let UNIQUE_ID = 0;
 class CardDrawers extends React.Component {
   constructor(props) {
     super(props);
+
     this._cardId = UNIQUE_ID++;
+
+    this.state = {
+      active: null
+    };
+  }
+
+  handleLinkClick(id) {
+    this.props.onClick && this.props.onClick();
+
+    this.setState(state => ({
+      active: state.active === id ? null : id
+    }));
   }
 
   render() {
+    const active = this.props.active || this.state.active;
```

Comment:
Are you sure it works the way you expect? When you click on the` CardToolbarLink` it will set `state.active = drawer.id` and even if your `active` prop is set explicitly to null, the state will take over.